### PR TITLE
Creation of Asinh for php

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create reference of the asinh function for php. Motivation still stems from the enthusiasm to contribute more to docs and help others.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/fmod/fmod.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/fmod/fmod.md:1]: You need to remove this file from this PR.

File: content/php/concepts/math-functions/terms/asinh/asinh.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The **`asinh()`** returns a floating point number which is the inverse hyperbolic sine value of a number passed to it as argument.
+
+## Syntax
+
+```pseudo
+asinh(value)
+```
+
+The `asinh()` function takes only one parameter, `value`. The value of this parameter must be in `radians`.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/asinh/asinh.md:25]:
```suggestion
The `asinh()` function takes only one parameter, `value`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz I did a first review and left some comments. Most important is you need to remove the fmod.md file from this PR.

File: content/php/concepts/math-functions/terms/fmod/fmod.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/fmod/fmod.md:1]: duly noted with thanks

Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/fmod/fmod.md:1]: just wonderin though

do i need to submit another PR

thanks

Comment (SSwiniarski) [reply content/php/concepts/math-functions/terms/fmod/fmod.md:1]: You shouldn't have to, this should resolve the conflict.

**Comment by arisdelacruz** [comment]
@SSwiniarski

I hope i got the message right. No needed move from my end as of the moment.

Thanks

**Comment by arisdelacruz** [comment]
i tried to resolve the branch conflict

so, i hope i did the right thing

thanks

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz @SSwiniarski I just pushed some minor edits. Looks good to merge

**Comment by SSwiniarski** [comment]
@arisdelacruz Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/math-functions/asinh

image

**Pull request merged**